rye/src/cli/publish.rs

```diff
@@ -45,6 +50,9 @@ pub struct Args {
     /// Continue uploading files if one already exists (only applies to repositories supporting this feature)
```

Comment:
CCing this for myself in case we decide to sneak this improvement into this PR https://github.com/astral-sh/rye/pull/831#discussion_r1510455906